#! /bin/sh /usr/share/dpatch/dpatch-run
## run-scripts-as-www-data.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: The scripts should be run with same context as "freepbx", this is 
## DP: not "asterisk" but "www-data"

@DPATCH@
diff -urNad freepbx-module-zapauto~/share/detect_zap freepbx-module-zapauto/share/detect_zap
--- freepbx-module-zapauto~/share/detect_zap	2007-04-18 15:56:10.000000000 +0300
+++ freepbx-module-zapauto/share/detect_zap	2007-04-18 15:57:14.000000000 +0300
@@ -54,8 +54,8 @@
 				
 set -e
 
-AST_USER=asterisk
-AST_GROUP=asterisk
+AST_USER=www-data
+AST_GROUP=www-data
 AST_SHELL=/bin/sh
 
 # find new HW, asterisk will be running after it, as zap2amp
